llvm/lib/IR/RepoDefinition.cpp

```diff
@@ -162,34 +171,31 @@ void ModuleHashGenerator::calculateGONumAndGOIMap(const Module &M) {
     ++NumFunctions;
   }
 #ifndef NDEBUG
-  for (auto &GI : GOIMap) {
-    LLVM_DEBUG(dbgs() << "\nGO Name:" << GI.first->getName() << "\n");
-    LLVM_DEBUG(GI.second.dump());
+  LLVM_DEBUG(dbgs() << "\nDump the GOs' information after converting the "
+                       "Contributions to Dependencies.\n");
+  for (auto &GO : M.global_objects()) {
+    if (GOIMap.count(&GO)) {
+      const auto GI = GOIMap[&GO];
```

Comment:
Shouldn’t this be a reference rather than a copy?

Reply:
Thanks, fixed in https://github.com/SNSystems/llvm-project-prepo/pull/127/commits/4a8756041970f24b02df38128f15c683126a7ebe.